exporter/exporterhelper/internal/persistent_queue.go

```diff
@@ -36,12 +35,19 @@ type persistentQueue struct {
 	storage    persistentStorage
 }
 
-// NewPersistentQueue creates a new queue backed by file storage; name parameter must be a unique value that identifies the queue
-func NewPersistentQueue(ctx context.Context, name string, capacity int, logger *zap.Logger, client storage.Client, unmarshaler RequestUnmarshaler) ProducerConsumerQueue {
-	return &persistentQueue{
+// PersistentQueueStartFunc defines a function that can be used to start the persistent queue
+type PersistentQueueStartFunc func(ctx context.Context, client storage.Client)
```

Comment:
why is this change necessary now and not before? if this can be "refactored" as a separate PR, please do so.